test/test_helper.rb

```diff
@@ -31,12 +31,16 @@ def requires_toxiproxy
     skip("Toxiproxy is not running, but was required for this test.") unless Toxiproxy.running?
   end
 
-  def assert_changed(object, attribute)
-    original = object.send(attribute)
+  def assert_changed(object, *attributes)
```

Comment:
This change allows to assert multiple changes with one `assert_changed` call as used later in `user_test` added test.